docs/lakehouse-iceberg-catalog.md

```diff
@@ -264,16 +264,21 @@ You can pass [Iceberg table properties](https://iceberg.apache.org/docs/1.5.2/co
 
 The Gravitino server doesn't allow passing the following reserved fields.
 
-| Configuration item              | Description                                             |
-|---------------------------------|---------------------------------------------------------|
-| `comment`                       | The table comment.                                      |
-| `creator`                       | The table creator.                                      |
-| `location`                      | Iceberg location for table storage.                     |
-| `current-snapshot-id`           | The snapshot represents the current state of the table. |
-| `cherry-pick-snapshot-id`       | Selecting a specific snapshot in a merge operation.     |
-| `sort-order`                    | Selecting a specific snapshot in a merge operation.     |
-| `identifier-fields`             | The identifier fields for defining the table.           |
-| `write.distribution-mode`       | Defines distribution of write data                      |
+| Configuration item        | Description                                                                          | Default value | Required | Since Version |
+|---------------------------|--------------------------------------------------------------------------------------|---------------|----------|---------------|
+| `comment`                 | The table comment, please use `comment` field in table meta instead.                 | None          | No       | 0.2.0         |
+| `creator`                 | The table creator.                                                                   | None          | No       | 0.2.0         |
+| `current-snapshot-id`     | The snapshot represents the current state of the table.                              | None          | No       | 0.2.0         |
+| `cherry-pick-snapshot-id` | Selecting a specific snapshot in a merge operation.                                  | None          | No       | 0.2.0         |
+| `sort-order`              | Iceberg table sort order, please use `SortOrder` in table meta instead.              | None          | No       | 0.2.0         |
+| `identifier-fields`       | The identifier fields for defining the table.                                        | None          | No       | 0.2.0         |
+| `write.distribution-mode` | Defines distribution of write data, please use `distribution` in table meta instead. | None          | No       | 0.2.0         |
```

Comment:
Why does this property have no default value?

Reply:
It doesn't have default property values to show up, you may mean Iceberg has a default value for `write.distribution-mode`, but we shouldn't depend on it.

Reply:
Yes. Different Iceberg table may have different default values, too. I thought this property is a property to create a catalog. If not, it's ok. https://iceberg.apache.org/docs/latest/spark-writes/#writing-distribution-modes

Reply:
maybe we could remote default value column for this table, because most properties is not managed by Gravitino, so default value not so much meaningful here, it just shows these properties are immutable.

Reply:
I removed `default value` and `required` column from the table, since this's seems not so useful for reserved properties. cc @mchades

Reply:
You can state that the default value is generated by Iceberg and provide the link.

Reply:
this seems complicated.

Reply:
> I removed `default value` and `required` column from the table, since this's seems not so useful for reserved properties. cc @mchades

It's ok for me. One Iceberg user like me will be confused about the different default value.

Reply:
I don't understand what you mean about `different default value`?